cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -304,6 +309,16 @@ func (s *CMServer) Run(_ []string) error {
 	}
 	pvRecycler.Run()
 
+	if s.EnablePersistentVolumeProvisioner {
+		if createableVolumePlugin := getCreatableVolumePluginForCloud(cloud, s); createableVolumePlugin != nil {
```

Comment:
I suppose the question is whether we'll have 0, 1, or Many provisioners in a cluster.  This implementation currently ties this feature to a cloud provider where we already have APIs for storage.  But the feature is also experimental.

For now, 1 cloud to 1 volume plugin.   They are mapped in plugins.go where the binary configures its plugins.